docs/Workshops/overview.md

```diff
@@ -0,0 +1,15 @@
+# Overview
+
+Research Computing Service (RCS) team at Imperial is trying to collaborate with Industrial partners/vendors to provide some training and workshops for the Imperial College community.
```

Comment:
```suggestion
The Research Computing Service (RCS) team at Imperial collaborates with industrial partners/vendors to provide training and workshops for the Imperial College community.
```

Reply:
I've suggested a rephrasing of that line.

Reply:
Thanks for the suggestion. I have accepted the same.